website/docs/features/file-uploads.mdx

```diff
@@ -116,3 +116,32 @@ const server = createServer({
 // Start the server and explore http://localhost:4000/graphql
 server.start()
 ```
+
+## Disabling Multipart Request Processing
+
+If you want to disable multipart request processing for some reason, you can pass `multipart: false` to prevent Yoga handling multipart requests;
```

Comment:
```suggestion
If you want to disable multipart request processing for some reason, you can pass `multipart: false` to prevent Yoga from handling multipart requests.
```